src/components/views/TablesEventNew/TablesEventNew.js

```diff
@@ -1,10 +1,94 @@
 import React from 'react';
-import styles from './TablesEventNew.scss';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import Button from '@material-ui/core/Button';
 
-const TablesEventNew = () => (
-  <div className={styles.component}>
-    <h2>TablesEventNew view</h2>
-  </div>
-);
+import styles from './TablesEventNew.module.scss';
 
-export default TablesEventNew;
+export default function TablesEventNew() {
```

Comment:
Ten komponent jest prawie identyczny jak  TableBookingNew. Unikaj duplikowani tego samego kodu. TO co takie samo wydziel do innego komponenetu i jego użyj